src/components/Product.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { ProductType } from '@/types/product';
+
+export default function Product({
+  id,
+  imageURL,
+  productName,
+  description,
+  price,
+}: ProductType) {
+  return (
+    <div className="flex w-[90%] flex-col gap-2 p-3">
+      <Link href={`/product/${id}`}>
```

Comment:
```suggestion
    <Link href={`/product/${id}`} className="flex w-[90%] flex-col gap-2 p-3"
```
Request: `Link`로 묶으면 좋을 것 같습니다.

Reply:
000d436057e8de60d89748cceade537a216e4f88

수정했습니다!